pkg/ops/iso.go

```diff
@@ -37,7 +37,7 @@ func GenISO(name, src, dst string, i schema.ISO) func(ctx context.Context) error
 		}
 
 		log.Info().Msgf("Generating iso '%s' from '%s' to '%s'", name, src, dst)
-
+		log.Info().Msg("ENKI")
```

Comment:
dev output? Otherwise that's just a technical detail, the user doesn't know (or care) what enki is.

Reply:
yes I was building aurora against the local enki and wanted to make sure I was calling the proper enki backend looool

Reply:
done!